dao-contracts/src/reputation.rs

```diff
@@ -157,6 +133,7 @@ impl ReputationContractInterface for ReputationContract {
         self.owner.ensure_owner();
         self.owner.change_ownership(owner);
         self.whitelist.add_to_whitelist(owner);
+        self.whitelist.remove_from_whitelist(caller());
```

Comment:
Why do we remove the previous owner from the whitelist?
Anyway, it took me a while to understand what's happening. If you assigned `caller()` to a variable and named it like `prev_owner`, I would instantly get it. 

CAUTION! 
A thoughts stream.
One more thing concerns me: this method invokes the `caller()` function twice, for the first time under the hood, inside the Owner module, and for the second time explicitly. I wonder whether it's a good approach to use `env` directly in the low-level modules. Maybe we should pass such info from top to bottom. E.g. only the contract layer can interact with the env (getting caller, block time, etc.). It would generate a bit more code because some argless function would be parametrized like the mentioned `ensure_owner()`, but it would make modules pure in return. Generally, concepts such as an env are shady, I never liked to have some god instances.